yt_dlp/extractor/cbs.py

```diff
@@ -187,6 +189,7 @@ class ParamountPressExpressIE(InfoExtractor):
             'thumbnail': r're:^https://.+\.jpg',
             'tags': [],
         },
+        'skip': 'geo-restricted',
```

Comment:
this extractor is not ThePlatform-related

Reply:
see above